docs/ML-Agents-Overview.md

```diff
@@ -594,18 +612,17 @@ learns, the ML-Agents Toolkit provides a way to randomly sample parameters of
 the environment during training. We refer to this approach as **Environment
 Parameter Randomization**. For those familiar with Reinforcement Learning
 research, this approach is based on the concept of Domain Randomization (you can
-read more about it [here](https://arxiv.org/abs/1703.06907)). By using parameter
-randomization during training, the agent can be better suited to adapt (with
+read more about it [here](https://arxiv.org/abs/1703.06907)). By using
```

Comment:
nit: just link "Domain Randomization" to the arxiv paper? (and remove "you can read more about it here")

Reply:
Done.